localstripe/resources.py

```diff
@@ -2361,9 +2360,16 @@ def _update(self, metadata=None, items=None, trial_end=None,
         except AssertionError:
             raise UserError(400, 'Bad request')
 
-        if items is None or len(items) != 1 or not items[0]['plan']:
+        if tax_percent is not None:
+            self.tax_percent = tax_percent
+
+        if items is None or len(items) != 1:
             raise UserError(500, 'Not implemented')
 
+        # If no plan specified in update request, we stay on the current one
+        if not items[0]['plan']:
+            items[0]['plan'] = self.items._list[0].plan.id
```

Comment:
```suggestion
            items[0]['plan'] = self.plan.id
```
will be more straightforward :+1: